crates/bevy_app/src/app_builder.rs

```diff
@@ -259,11 +363,47 @@ impl AppBuilder {
         self
     }
 
+    /// Sets the main runner loop function for bevy application
+    ///
+    /// Usually the main loop is handled by bevy integrated plugins (`winit`), but
+    /// in some cases one wants to implement an own main loop.
+    ///
+    /// This method sets the main loop function. Overwrites previous runner.
+    ///
+    /// You should call `app.update()` in the runner to trigger the bevy ecs system.
```

Comment:
You need to call `app.init()` once at the start to run all startup systems.

Reply:
Ok. That might confuse users, so probably best to remove the last line.